edgraph/access_ee.go

```diff
@@ -1274,17 +1274,17 @@ func groupFilter(groupIds []string) *gql.FilterTree {
 }
 
 /*
- addUserFilterToFilter makes sure that user can't misue filters to access other user's info.
- If the *filter* have type(dgraph.type.Group) or type(dgraph.type.User) functions,
- it generate a *newFilter* with function like eq(dgraph.xid, userId) or eq(dgraph.xid,groupId...)
- and return a filter of the form
-
-		&gql.FilterTree{
-			Op: "AND",
-			Child: []gql.FilterTree{
-				{filter, newFilter}
+	 addUserFilterToFilter makes sure that user can't misue filters to access other user's info.
+	 If the *filter* have type(dgraph.type.Group) or type(dgraph.type.User) functions,
+	 it generate a *newFilter* with function like eq(dgraph.xid, userId) or eq(dgraph.xid,groupId...)
+	 and return a filter of the form
+
+			&gql.FilterTree{
+				Op: "AND",
+				Child: []gql.FilterTree{
+					{filter, newFilter}
+				}
```

Comment:
[nitpick ~ no need to fix] I notice the `tabbing` & related aesthetic changes coming in from your PRs - possibly from your IDE formatter. I personally think we should avoid introducing these changes, when the PR is targeting an older branch.

Reply:
So the original change went into main and it had the formatting changes like you mentioned. Since this PR is only a cherry-pick, I brought in all the changes from the other merge. Fixing (removing) these would require changes on top of cherry-pick PR.